chapter_builders-guide/init-param.md

```diff
@@ -270,24 +338,44 @@ net(X)
 print(net.layers[1].weights[0])
 ```
 
+```{.python .input}
+%%tab jax
+def my_init(key, shape, dtype=jnp.float_):
+    data = jax.random.uniform(key, shape, minval=-10, maxval=10)
+    return data * (jnp.abs(data) >= 5)
+
+net = nn.Sequential([nn.Dense(8, kernel_init=my_init), nn.relu, nn.Dense(1)])
+params = net.init(jax.random.PRNGKey(d2l.get_seed()), X)
+print(params['params']['layers_0']['kernel'][:, :2])
+```
+
+:begin_tab:`mxnet, pytorch, tensorflow`
 Note that we always have the option
 of setting parameters directly.
+:end_tab:
+
+:begin_tab:`jax`
+When initializing parameters in JAX and Flax, the parameters dict returned
```

Comment:
Use proper English in main text.
parameters dict -> the dictionary of parameters, or the parameter dictionary